src/Microsoft.OpenApi/Any/IOpenApiAny.cs

```diff
@@ -1,14 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. 
 
+using System;
 using Microsoft.OpenApi.Interfaces;
 
 namespace Microsoft.OpenApi.Any
 {
     /// <summary>
     /// Base interface for all the types that represent Open API Any.
     /// </summary>
-    public interface IOpenApiAny : IOpenApiElement, IOpenApiExtension
+    public interface IOpenApiAny : IOpenApiElement, IOpenApiExtension, ICloneable
```

Comment:
pushing more constraints on an interface is a breaking change, this needs to be reverted